hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/replicatedmap/ReplicatedMapGetMessageTask.java

```diff
@@ -51,15 +52,15 @@ protected Operation prepareOperation() {
 
     @Override
     protected void beforeProcess() {
-        startTimeNanos = System.nanoTime();
+        startTime = Timer.millis();
     }
 
     @Override
     protected Object processResponseBeforeSending(Object response) {
         ReplicatedMapService replicatedMapService = getService(ReplicatedMapService.SERVICE_NAME);
         if (replicatedMapService.getReplicatedMapConfig(parameters.name).isStatisticsEnabled()) {
             LocalReplicatedMapStatsImpl stats = replicatedMapService.getLocalReplicatedMapStatsImpl(parameters.name);
-            stats.incrementGets(System.nanoTime() - startTimeNanos);
+            stats.incrementGets(Timer.elapsedSince(startTime));
```

Comment:
There are multiple problems:

1: the content of the stats.get field is interpreted as nanoseconds. But you are updating in miliseconds, meaning that the value you are writing is 10^6 times less than it should be.

2: you are loosing granularity. So you can't measure a call that has a submilisecond latency and on a reasonably decent cluster a remote call is in the 100/200us range (and sometimes a hell of a lot less).

3: the time unit returned from the elapsedSince method isn't clear. This makes it very easy to make unit errors.

1: the original precision of the get was in nanoseconds. But now is in miliseconds.
2: it isn't clear from the elapsedSince what kind of time unit is returned. 
3: reduced granularity of the stats because you can't track submilisecond latency.